app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
```

Comment:
__[rubocop]__
* Module has too many lines. [216/100]
* Missing top-level module documentation comment.
* Use nested module/class definitions instead of compact style.